### DISCUSSION TITLE
Sound every houre

### DISCUSSION THREAD

**Discussion by greycos** [CircuitSetup/Time-Circuits-Display#13] [Ideas]
Hi first of all thx for the great product.

How about implement a sound every hour?
So if it turns 01:00  it should play a [Sound](https://www.zedge.net/ringtone/ecf9636f-7d1e-3403-abd8-c31d6acce059) like this one.

**Comment by CircuitSetup** [comment]
FYI, this is implemented in [v2 of the software](https://github.com/CircuitSetup/Time-Circuits-Display/releases)

**Comment by liamamiller** [comment]
Were you able to figure this out? I've uploaded my sound files from an SD card but don't see a way to assign a sound to a particular hour. I'm wanting the BTTF chimes to play when the time is 11:11 :)

**Comment by greycos** [comment]
Hi yeah the normal every hour is working, but i think the only way that you can play the sound on a particular hour you have to use the Alarm feature. I don't know if you can change the biep sound with yours, try to substitute it with your Sound, renaming it to alarm.mp3.
but like i sad dont know if its working...

**Comment by liamamiller** [comment]
Hmm. I see the option "Alarm base is real present time" but don't see an option to "sound on the hour" which is in the wiki. I'll have to keep playing with it.

**Comment by greycos** [comment]
Just that i understand you right you want a alarm, let's say every day at 11:11pm so you only can do it to set the alarm at this time. Only problem you hear piepiepie. On the other hand if you want it every hour like a clock tower or KuckKuck clock like in the movie you have to rename your mp3 sound file in  hour-00.mp3 to hour-23.mp3 so lats say you want play a gong sound on 11pm. you rename your mp3 file with a gong, to hour-23.mp3. so on 11pm it should play your file.
Like you see its only working for every full hour like a clock. but if you want it on 11:11pm the only way is 1st to programm it your self or 2nd you abusing the alarm clock that you set trough the keypad menu. 
The ReadMe also says :

The provided audio files ("sound-pack") are, after proper installation, integral part of the firmware and stored in the device's flash memory.
These sounds can be substituted by your own sound files on a FAT32-formatted SD card. These files will be played back directly from the SD card during operation, so the SD card has to remain in the slot. The built-in Audio file installer cannot be used to replace default sounds in the device's flash memory with custom sounds.
Your replacements need to be put in the root (top-most) directory of the SD card, be in mp3 format (128kbps max) and named as follows:

"alarm.mp3". Played when the alarm sounds.
 
hope that helps?

**Reply by liamamiller** [reply]
Helps tremendously! Thank you. 

I looked through the Wiki again and didn't see anything about naming files hour-xx.mp3 -- Can you point me in a direction where that information is?

**Reply by liamamiller** [reply]
Scratch that - I found the link!
